app/serializers/articles_index_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class ArticlesIndexSerializer < ActiveModel::Serializer
+  attributes :id, :title, :teaser, :date
+
+  def date
+    object.created_at.strftime('%F')
```

Comment:
Good. There is a potential problem with this solution, but that will manifest itself later if ever.